lib/models/task.dart

```diff
@@ -203,11 +207,52 @@ class Task {
   String toString() {
     return "Task($name, $id, $recurrenceRules)";
   }
+  
+  String toDetailedString() {
+    return "Task($name, $id, $description, $completed, $location, $color, $recurrenceRules, $tags, $timeStart, $timeDue, $timeCurrent, $timeCreated, $timeModified)";
+  }
 
   @override
   bool operator ==(Object other) {
```

Comment:
How come you only want to check name and ID? Is this not gonna be used to see if two separate tasks have the same fields?

Reply:
It got cut off by github
![image](https://github.com/Scrumbags115/plannertarium/assets/130195350/92773df1-2865-4765-95de-1ec5ca50fd4f)

Reply:
bruh. silly github!